lib/workers.js

```diff
@@ -0,0 +1,222 @@
+const { EventEmitter } = require('events');
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [File has too many classes (2). Maximum allowed is 1.](https://app.codacy.com/manual/DavertMik/CodeceptJS/pullRequest?prid=4759786)